Tuesdays/Week-Class/Inheratance/Assets/Scripts/Trap.cs

```diff
@@ -30,7 +34,8 @@ public abstract class Trap : MonoBehaviour, IDamagable{
 		//Debug.Log(collision.collider.gameObject.layer);
 		//Debug.Log(whatIDamage & (1 << collision.collider.gameObject.layer));
 		if ((whatIDamage & (1 << collision.collider.GetComponent<PlayerController>().CurrCharacter.gameObject.layer)) != 0){
```

Comment:
hmmm, I not sure it does what you think it does.
that is a binary operation that shifts a 1 as a bit to the left equal to the game object layer as an int